libs/cohere/langchain_cohere/chat_models.py

```diff
@@ -52,6 +53,40 @@
 from langchain_cohere.llms import BaseCohere
 
 
+def _get_tool_results(messages: List[BaseMessage]) -> List[Dict[str, Any]]:
+    """Get tool_results from messages."""
+    tool_results = []
+    for message in messages:
+        if isinstance(message, ToolMessage):
+            tool_message = message
+            previous_ai_msgs = [
+                message
+                for message in messages[:-1]
+                if isinstance(message, AIMessage) and message.tool_calls
+            ]
+            if previous_ai_msgs:
+                previous_ai_msg = previous_ai_msgs[-1]
+                tool_message_name: Optional[str] = None
+                if tool_message.name:
+                    tool_message_name = tool_message.name
+                else:
+                    tool_message_name = tool_message.additional_kwargs.get("name")
+                tool_results.extend(
+                    [
+                        {
+                            "call": ToolCall(
+                                name=lc_tool_call["name"],
+                                parameters=lc_tool_call["args"],
+                            ),
+                            "outputs": [{"answer": tool_message.content}],
```

Comment:
Could you use `convert_to_document`([link](https://github.com/langchain-ai/langchain-cohere/blob/main/libs/cohere/langchain_cohere/react_multi_hop/prompt.py#L140)) instead? I'd be more than happy to extract/rename it - we'll keep it updated with the best way to convert `Any` (including strings) into prompt content.

```suggestion 
                            "outputs": [{"output": tool_message.content}],
```